docs/development.rst

```diff
@@ -37,12 +37,68 @@ To run a single test, use this following command, replace the test case name at
     make test TESTS=tests/test_api.py::TestAPI::test_error_unencrypted_reply
 
 
-To test against a live development server, you will need to run the SecureDrop developent
-container from the main SecureDrop repository on your host. This can be done via ``make -C securedrop dev``.
+To test against a live development server, you will need to run the SecureDrop
+developent container from the main SecureDrop repository on your host. This
+can be done via ``NUM_SOURCES=5 make -C securedrop dev``.
+
+In this repo, comment out the ``@vcr`` decorator of the ``setUp`` method in
+`test_api.py` and execute which ever tests you want to run. If you want to
+re-run all tests against the API, remove all the ``.yml`` files in the
+``data`` directory. 
+
+
+Gererating test data for `APIProxy`
+-----------------------------------
+
+
+To test or to generate new test data file for the `APIProxy` class in
+`test_apiproxy.py` file, you will have to setup
+[QubesOS](https://qubes-os.org) system.
+
+There should be one VM (let us call it `sd-journalist`), where we can run
+latest securedrop server code from the development branch using
+``NUM_SOURCES=5 make -C securedrop dev`` command. The same VM should also have
+`securedrop-proxy` project installed, either from the source by hand or using
+the latest Debian package from the FPF repository.
+
+Below is an example configuration for proxy `/etc/sd-proxy.yaml`
+
+::
```

Comment:
For explicitness, I like putting `:: code-style: none` on these blocks (there's a few cases of this).

Reply:
This is the default behavior of literal blocks in Sphinx. https://www.sphinx-doc.org/en/stable/usage/restructuredtext/basics.html#literal-blocks You can add `code-block` when you are adding a particular programming language example.